python_modules/dagster/dagster/_core/definitions/time_window_partitions.py

```diff
@@ -2171,6 +2171,10 @@ def with_partitions_def(
     def __repr__(self) -> str:
         return f"TimeWindowPartitionsSubset({self.get_partition_key_ranges(self.partitions_def)})"
 
+    @property
+    def is_serializable(self) -> bool:
```

Comment:
Rather than needing to add a property that needs to match up `@whitelist_for_serdes`, would there be some way to inspect the object / class to determine whether it's decorated with `@whitelist_for_serdes`? Presumably we need to do that check already somewhere in the serdes code.

Reply:
I added a method that checks for this